scripts/push_example

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+readonly SCRIPTS_DIR="$(dirname "$0")"
+# shellcheck source=scripts/common
+source "${SCRIPTS_DIR}/common"
+
+while getopts "e:h" opt; do
+  case "${opt}" in
+    h)
+      echo -e "Usage: ${0} [-h] -e EXAMPLE
+
+Save example modules in Google Cloud Storage.
+
+Options:
+  -e    Example application name (required)
+  -h    Print Help (this message) and exit"
+      exit 0;;
+    e)
+      readonly EXAMPLE="${OPTARG}";;
+    *)
+      echo "Invalid argument: ${OPTARG}"
+      exit 1;;
+  esac
+done
```

Comment:
This seems excessive :) The error message below should be enough I think?

Reply:
Just making it similar to other scripts that use `-e` for referring to an example.

Reply:
I still suggest we use -e for the example.. but seeing that this has only one way to use it, I don't think we need all the `getops` code for it. If you remove this and it's not used correctly, it will still print the correct message

Reply:
I have only one word to add, and I am not even going to actually mention it :) (it starts with R)

Reply:
I can just make it so that `${1}` will be `-e` and `${2}` will be a parameter.
But I think it will be less readable.

Reply:
let's keep as it is for now, and think what to do about it in `runner` perhaps in the future.